apps/meteor/server/lib/roles/addUserRoles.ts

```diff
@@ -14,6 +14,11 @@ export const addUserRolesAsync = async (userId: IUser['_id'], roleIds: IRole['_i
 		throw new MeteorError('error-invalid-user', 'Invalid user');
 	}
 
+	if (!roleIds?.length) {
```

Comment:
The purpose of this function is only to add new roles to users (or to subscriptions), it should never remove any role that may already be assigned.